modules/nf-core/vg/construct/main.nf

```diff
@@ -0,0 +1,68 @@
+process VG_CONSTRUCT {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::vg=1.45.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/vg:1.45.0--h9ee0642_0':
+        'quay.io/biocontainers/vg:1.45.0--h9ee0642_0' }"
+
+    input:
+    tuple val(meta), path(vcfs), path(tbis), path(msa), path(insertions_fasta)
+    tuple val(meta2), path(fasta)
```

Comment:
Could we maybe just rename `fasta` to `reference`, since that's what the parameter is called? Would make the module easier to parse :-)

Does the fasta index need to be a separate channel? Seems like it relates closely to the fasta.

Reply:
I'm following the way it's done in other modules. I think the fasta and its index should go together too, but I don't really want to do it different than in other modules for ease of development.

Reply:
Fair enough!